pytorch/buildspec-1-5.yml

```diff
@@ -0,0 +1,99 @@
+  account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+  region: &REGION <set-$REGION-in-environment>
+  framework: &FRAMEWORK pytorch
+  version: &VERSION 1.5.0
```

Comment:
Our main buildspec file will be for 1.5 only. 
Changes in 1.4 will far and fewer than PT1.5.
Please create a sperate buildspce for PT1.4 not PT1.5

Reply:
What I want to do is to use it as a one time release purpose file. Will recover to PT1.5 once PT 1.4 patch release done.
Which method do you prefer? To do modification on root buildspec.yml? It also requires a recover operation

Reply:
Let's retain 1.5 buildspecs.

Reply:
Thanks for confirming.